src/Avalonia.Controls/Repeater/ItemsRepeater.cs

```diff
@@ -61,8 +64,9 @@ public class ItemsRepeater : Panel
         private readonly ViewportManager _viewportManager;
         private IEnumerable _items;
         private VirtualizingLayoutContext _layoutContext;
-        private NotifyCollectionChangedEventArgs _processingItemsSourceChange;
+        private EventHandler<ChildIndexChangedEventArgs> _childIndexChanged;
         private bool _isLayoutInProgress;
+        private NotifyCollectionChangedEventArgs _processingItemsSourceChange;
         private ItemsRepeaterElementPreparedEventArgs _elementPreparedArgs;
```

Comment:
@grokys you reused single instance of event args per ItemsRepeater for its events. Do you recommend do it with new event from IChildIndexProvider?

I thought we don't really want to do it, because developer can keep reference for older event args on his side, and expect them to be different object. Although it's hard to imagine reasoning of that.

Reply:
It was done like that because that's how winui does it. However from later experiments recycling e.g. property changed events, I did not find much perf advantage. Might be that it's more necessary when running on winrt.